website/components/side-by-side/style.module.css

```diff
@@ -0,0 +1,62 @@
+.sideBySide {
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (--large) {
+    flex-wrap: nowrap;
+  }
+
+  & .sideWrapper {
```

Comment:
I extracted some of the styles from g-grid-container and applied it to work for this section